File: spring25/oop labs/JAVA/oop_lab3/task1.java
// import java.util.Scanner;

// public class task1{
//     public static void main(String[] args){
//         Scanner sc = new Scanner(System.in);
//         System.out.println("Enter the size of array: ");
//         int size = sc.nextInt();

//         if (size < 2){
//             System.out.println("Array size should be greater than 1");
//             System.exit(0);
//         }

//         int[] arr = new int[size];
//         System.out.println("Enter the elements of array: ");

//         for(int i = 0; i < size; i++){
//             arr[i] = sc.nextInt();
//         }

//         int temp = arr[0];
//         arr[0] = arr[size-1];
//         arr[size-1] = temp;

//         System.out.println("After swapping: ");
//         for(int i = 0; i < size; i++){
//             System.out.println(arr[i]);
//         }
//         sc.close();
// }
// }

File: fall25/dsa/lab2/task1.cpp
#include <iostream>

using namespace std;

int main()
{
    int size;
    cout << "Enter the size of array: ";
    cin >> size;

    int *arr = new int[size];

    for (int i = 0; i < size; i++)
    {
        arr[i] = i + 1;
        cout << "arr[" << i << "] = " << arr[i] << endl;
    }

    delete[] arr;

    return 0;
}